### ISSUE TITLE
Regression after version 1.6.9 spring.webflux.base-path repeats on page reload

### ISSUE THREAD

**Issue by vladgon** [springdoc/springdoc-openapi#1818]
**When using spring.webflux.base-path, the prefix multiply on each page reload **
- This issue appears when server is explicitly set e.g. localhost and spring.webflux.base-path is set. This is a regression still not fixed in 1.6.11, works fine in 1.6.9 
tested with Springboot version 2.7.3

**To Reproduce**
- Set the spring.webflux.base-path and reload the page, the operation path grows with each reload

**Comment by bnasslahsen** [comment]
@vladgon,

Shoud be same root cause as https://github.com/springdoc/springdoc-openapi/issues/1820

**Comment by vladgon** [comment]
in this case the operation path grows on each refresh e.g.
prefix/op/get
prefix/prefix/op/get
prefix/prefix/prefix/op/get

**Comment by bnasslahsen** [comment]
@vladgon,

Not reproducible.
Are you able to Provide a [Minimal, Reproducible Example - with HelloController](https://stackoverflow.com/help/minimal-reproducible-example) that reproduces the problem ?

**Comment by vladgon** [comment]
The issue is in OpenApiCustomiser is invoked on every page refresh after version 1.6.9 and before was invoked on startup only once.

Please see sample code in https://github.com/vladgon/SpringDoc

 

 @Bean
    OpenApiCustomiser openApiCustomiser(@Value("${spring.webflux.base-path}") String base) {
        return openApi -> openApi.setPaths(openApi.getPaths()
                                                  .entrySet()
                                                  .stream()
                                                  .reduce(new Paths(),
                                                          (paths, item) -> paths.addPathItem(
                                                                  java.nio.file.Paths.get(base, item.getKey())
                                                                                     .toString(),
                                                                  item.getValue()),
                                                          (paths, paths2) -> paths));
    }

**Comment by bnasslahsen** [comment]
@vladgon,

Good that you have added the sample. 
It's confirmed that it's the same root cause: `OpenApiCustomiser`...
So it's duplicate of https://github.com/springdoc/springdoc-openapi/issues/1820.

Don't hesitate to test it with the [latest SNAPSHOT ](https://springdoc.org/#how-can-i-use-the-last-springdoc-openapi-snapshot)and provide your feedback.
